package.json

```diff
@@ -5,9 +5,9 @@
   "main": "./lib/index.js",
   "scripts": {
     "lint": "eslint .",
-    "prepublish": "babel --out-dir ./lib ./src",
+    "prepublish": "NODE_ENV=development babel --out-dir ./lib ./src",
```

Comment:
can you try removing it and trying it out on your machine? It looks like it is required for the preset update, it's pretty harmless, and I've tested it and verified the output code works.